doc/bspwm.1

```diff
@@ -2,12 +2,12 @@
 .\"     Title: bspwm
 .\"    Author: [see the "Author" section]
 .\" Generator: DocBook XSL Stylesheets vsnapshot <http://docbook.sf.net/>
-.\"      Date: 04/05/2023
+.\"      Date: 01/01/1980
 .\"    Manual: Bspwm Manual
-.\"    Source: Bspwm 0.9.10-49-gc58c67a
+.\"    Source: Bspwm 0.9.10
 .\"  Language: English
 .\"
-.TH "BSPWM" "1" "04/05/2023" "Bspwm 0\&.9\&.10\-49\-gc58c67a" "Bspwm Manual"
+.TH "BSPWM" "1" "01/01/1980" "Bspwm 0\&.9\&.10" "Bspwm Manual"
```

Comment:
That looks fishy :)

Reply:
The act of running "make doc" generated that change. I could always revert that line.